examples/api/example-skipIsolated.ts

```diff
@@ -0,0 +1,14 @@
+import {detectClones} from "jscpd";
+
+(async () => {
+  const clones = await detectClones({
+    path: [
+      __dirname + '/../fixtures'
+    ],
+    skipIsolated: [
+      ['packages/businessA', 'packages/businessB', 'packages/businessC'],
```

Comment:
Thank you for the PR, it is a brilliant idea!

May I suggest some changes, we have a similar option named `--skipLocal` for show duplications across the folders, the `--skipLocal` used `path` array from options to define folders, I guess we should use the same way here, what do you think?

The result will be like:
```typescript
(async () => {
  const clones = await detectClones({
    path: [
      'packages/businessA', 
      'packages/businessB', 
      'packages/businessC'
    ],
    skipIsolated: true,
    silent: true
  });
  console.log(clones);
})()
```

Reply:
@kucherenko Thank you for your advice. Sorry, I didn't describe it clearly.

--skipIsolated means that submodules (packages/businessA, packages/businessB etc) can be isolated from each other when the detect path is the monorepo home project. And, other folders still want to be included in the clone code statistics normally.

```js
(async () => {
  const clones = await detectClones({
    path: ['./'], // e.g. detect clones from packages/businessA + globals + libs
    skipIsolated: [
        [ 'packages/businessA', 'packages/businessB', 'packages/businessC'],
    ],
    silent: true
  });
  console.log(clones);
})()
```

Reply:
ok, sounds good, will merge the PR and publish new version